packages/ui-components/src/components/tooltip/readme.md

```diff
@@ -75,13 +75,6 @@ export const TagLetterExample: React.FC = () => (
 | `"content"`   | The tooltip content.   |
 
 
-## CSS Custom Properties
```

Comment:
check this documentation removal. It seems this property continues to exist in our scss.